docs/man/rpm-setup-autosign.1.scd

```diff
@@ -0,0 +1,47 @@
+RPM-SETUP-AUTOSIGN(1)
+
+# NAME
+*rpm-setup-autosign* - Set up automatic signing for rpmbuild
+
+# SYNOPSIS
+*rpm-setup-autosign* [options]
+
+# DESCRIPTION
+*rpm-setup-autosign* is used to set up automatic signing from *rpmbuild*(1).
+It generates a user- and host-specific, passwordless OpenPGP key,
+configures *rpmbuild*(1) to use that key and exports the public key
+(aka certificake) for importing to the persistent *rpm*(8) keyring.
+
+The purpose of automatic signing is to make testing local builds
+painless. For distributing packages, it's recommended to use a separated
+signing account that cannot be compromised by a build.
+
+# ARGUMENTS
+*rpm-setup-autosign* takes no arguments.
+
+# OPTIONS
+*-p* <*gpg*|*sq*>,
+*--prog* <*gpg*|*sq*>
+	Specify the signing program to use: GnuPG or sequoia-sq.
```

Comment:
This is an unusual stylization.  Perhaps: `Sequoia PGP's sq`

Reply:
Oh yup, it seemed and felt wrong, just couldn't find any example of a similar use.